Source/SIMPLib/DataContainers/DataArrayProxy.cpp

```diff
@@ -324,3 +324,23 @@ void DataArrayProxy::updatePath(DataArrayPath::RenameType renamePath)
     path = newPath.serialize();
   }
 }
+
+uint8_t DataArrayProxy::getFlag() const
+{
+	return flag;
+}
+
+void DataArrayProxy::setFlag(uint8_t newFlag)
+{
+	flag = newFlag;
```

Comment:
Indentation is still off for getFlag and setFlag.  Comment blocks are missing above method definitions between getFlag and getName